src/main/java/im/toduck/domain/concentration/domain/usecase/ConcentrationUseCase.java

```diff
@@ -0,0 +1,49 @@
+package im.toduck.domain.concentration.domain.usecase;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import im.toduck.domain.concentration.domain.service.ConcentrationService;
+import im.toduck.domain.concentration.persistence.entity.Concentration;
+import im.toduck.domain.concentration.presentation.dto.request.ConcentrationRequest;
+import im.toduck.domain.concentration.presentation.dto.response.ConcentrationResponse;
+import im.toduck.domain.concentration.presentation.dto.response.ConcentrationSaveResponse;
+import im.toduck.domain.user.domain.service.UserService;
+import im.toduck.domain.user.persistence.entity.User;
+import im.toduck.global.annotation.UseCase;
+import im.toduck.global.exception.CommonException;
+import im.toduck.global.exception.ExceptionCode;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@UseCase
+@RequiredArgsConstructor
+public class ConcentrationUseCase {
+	private final UserService userService;
+	private final ConcentrationService concentrationService;
+
+	@Transactional
+	public ConcentrationSaveResponse saveConcentration(Long userId, ConcentrationRequest request) {
+		User user = userService.getUserById(userId)
+			.orElseThrow(() -> CommonException.from(ExceptionCode.NOT_FOUND_USER));
+
+		Concentration concentration = concentrationService.saveConcentration(user, request);
+
+		log.info("집중 저장 - UserId: {}, ConcentrationId: {}", userId, concentration.getId());
+		return new ConcentrationSaveResponse(concentration.getId());
+	}
+
+	@Transactional
+	public List<ConcentrationResponse> getMonthlyConcentration(Long userId, String yearMonth) {
+		User user = userService.getUserById(userId)
+			.orElseThrow(() -> CommonException.from(ExceptionCode.NOT_FOUND_USER));
+
+		List<Concentration> concentrations = concentrationService.getMonthlyConcentration(user, yearMonth);
+
+		return concentrations.stream()
+			.map(ConcentrationResponse::fromEntity)
+			.collect(Collectors.toList());
```

Comment:
p5: 
```suggestion
			.toList();
```
로 간소화 하는 것은 어떻게 생각하시나요??

Reply:
수정했습니다